devel/Dockerfile.template

```diff
@@ -21,16 +21,12 @@ RUN apt-get update -qq \
 	&& apt-get clean \
 	&& rm -f /var/lib/apt/lists/*_dists_*
 
-# Don't install any packages from bintray now. Error: "Insufficient
-# information available to perform this download securely".
-#
-# RUN (echo "deb https://dl.bintray.com/laarid/main @IMAGE_SUITE@ main" | tee /etc/apt/sources.list.d/bintray.list) \
-# 	&& apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 379CE192D401AB61 \
-# 	&& apt-get update -qq \
-# 	&& apt-get install --no-install-recommends -y --allow-unauthenticated
-# 		libbionic0-dev \
-# 	&& apt-get clean \
-# 	&& rm -f /var/lib/apt/lists/*_dists_*
+RUN (echo "deb [allow-insecure=yes] https://dl.bintray.com/laarid/main @IMAGE_SUITE@ main" | tee /etc/apt/sources.list.d/bintray.list) \
+	&& apt-get update -qq \
+	&& apt-get install --no-install-recommends -y --allow-unauthenticated \
+		laarid-devel \
```

Comment:
I think I should not install laarid-devel here. Or I won't be able to catch dependency failure in all package CI tests.